lib/routes/utilities/pushReplicationMetric.js

```diff
@@ -0,0 +1,35 @@
+const assert = require('assert');
+
+const { ObjectMD } = require('arsenal').models;
+const { pushMetric } = require('../../utapi/utilities');
+
+function shouldPushMetric(prevObjectMd, newObjectMD) {
```

Comment:
just for the sake of consistency and least surprise, you might want to use `MD` suffix on both variables (case difference here)

Reply:
I agree, updated.